### ISSUE TITLE
Alter msd to jpeg preprocessing to generate binary labels

### ISSUE THREAD

**Issue by NiWaRe** [TUM-AIMED/PrivateSegmentation#7]
**Description**
For now in `data/MSD/preprocessing/data_loader.py` even with `anti_aliasing=True` the labels aren't binary, after normalization to a range of [0, 1] they still contain other values besides {0, 1}. 

**Current fix**
`lambda x: np.where(
          x > 0.0,
          torch.ones_like(x),
          torch.zeros_like(
              x
          )
      )`
See in [here](https://github.com/TUM-AIMED/PrivateSegmentation/blob/ea7a96df8f5fdf7cd23357c107c3ebdd043ab01a/train.py#L239) in train.py. 
Once this is fixed, please also change the respective code in utils.py for the federated dataloading. 

**Reproduce/test:** 
The pictures have been loaded in with `PIL.Image.open` and converted to an array with numpy `np.array(img, dtype=np.float32)/255` (also with `dtype=np.uint8`). 
<be>
The result has been checked with: `(arr<1.0).sum() - (arr==0.).sum()`. E.g. for 0.jpg in train/ this results in 830 (should be 0)
